### PR TITLE
Test for invalid ampdoc dependencies

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: test/functional/test-extensions-deps.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (erwinmombay) [inline test/functional/test-extensions-deps.js]: 2016

**Comment by erwinmombay** [comment]
LGTM. 1 nit

**Comment by mrjoro** [comment]
This was approved; any reason not to merge it?

**Comment by dvoytenko** [comment]
@mrjoro I don't like the fact that we have to import all extensions in this test. Further, we started to walk away from the side-effect registration.

**Pull request closed without merging**